autopush/logging.py

```diff
@@ -113,12 +113,17 @@ def __call__(self, event):
 
     def raven_log(self, event):
         f = event["log_failure"]
-        stack = extra = None
+        stack = None
```

Comment:
This is stack, we include it in the sentry report, but its not the same 'stack' as the 'stack' we put into the client_info dict later. That seems a little confusing.